src/main/java/edu/wpi/grip/ui/preview/SocketPreviewView.java

```diff
@@ -17,21 +12,18 @@
  */
 public abstract class SocketPreviewView<T> extends TitledPane {
     private final EventBus eventBus;
-    private final Socket<T> socket;
-    private final ObjectProperty<T> valueProperty;
+    private final OutputSocket<T> socket;
 
     /**
      * @param eventBus The EventBus used by the application
      * @param socket   An output socket to preview
      */
-    public SocketPreviewView(EventBus eventBus, Socket<T> socket) {
+    public SocketPreviewView(EventBus eventBus, OutputSocket<T> socket) {
```

Comment:
You really don't need to pass the output socket to this abstract class.
It really doesn't need it. This class doesn't actually do anything with it.